README.md

```diff
@@ -4,19 +4,34 @@ This project is the seed for CircleCI's new command-line application.
 
 ## Getting Started
 
-### 1. Get the latest binary
+### From Scratch
 
-Download the [latest release](https://github.com/CircleCI-Public/circleci-cli/releases/latest) from GitHub for your operating system. If you're on a Mac, this would be `circleci-cli_0.1.XXX_darwin_amd64.tar.gz`.
+If you haven't already installed `circleci` on your machine, run the following command:
 
-### 2. Put the binary in your $PATH
+```
+curl -o /usr/local/bin/circleci \
+  https://raw.githubusercontent.com/zzak/local-cli/switch/circleci.sh && \
+  chmod +x /usr/local/bin/circleci
+```
+
+The CLI, `circleci`, is downloaded to the `/usr/local/bin` directory.
+
+If you do not have write permissions for `/usr/local/bin`, you might need to run the above commands with `sudo`.
+
+### Upgrade from existing CLI
```

Comment:
We should probably put this first, to avoid confusion (especially during the preview period)